web/impact/templates/directory.html

```diff
@@ -0,0 +1,17 @@
+{% load staticfiles %}
```

Comment:
I'd actually rather not have this template and use the html file generated by the `New-Mentor-Directory` but couldn't work that out.

Reply:
This template can contain whatever we want. Can you say more about what HTML you want here and what stopped you?

Reply:
yea - the way i had this in my head is that we would merge the contents of the new mentor directory app into impact and subsequently get rid of the New Mentor Directory repo (currently this depends on having the repos side by side but im not sure how this plays into our deploys (how does the static resource get generated by ECS deployments.)

Reply:
I see - ok so the built assets are packaged into the image as a volume. i think thats sufficient for now.